docs/pravega-concepts.md

```diff
@@ -304,18 +261,22 @@ as scale events occur over time.
 
 ### Ordering Guarantees
 
+A stream comprises a set of segments that can change over time. Segments that overlap in their area of keyspace have a defined order.
 
-Pravega provides the following ordering guarantees:
+An event written to a stream is written to a single segment and it is totally ordered with respect to the events of that segment. The existance and position of an event within a segment is strongly consistant.
 
-1.  Events with the same Routing Key are consumed in the order they were
-    written.
+Readers can be assigned multiple parallel segments (From different parts of keyspace). A reader reading from multiple segments will interleave the events of the segments, but the order of events per segment respects the one of the segment. Specifically, if s is a segment, events e1 and e2 of s are such that e1 precedes e2, and a reader reads both e1 and e2, then the reader will read e1 before e2.
```

Comment:
You may want to use subscript here for the events. If I remember correctly, it should be something like `e~1`.